approximate_baseline/approximateBandpasses.py

```diff
@@ -0,0 +1,35 @@
+import os
+import numpy as np
+from lsst.utils import getPackageDir
```

Comment:
How  about using `os.environ`? I actually never understood why we needed `get_PackageDir` rather than use the environment variable. So that may be an issue.